question-answering-app/data-loader/main.py

```diff
@@ -0,0 +1,112 @@
+"""
+ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+
+ WSO2 LLC. licenses this file to you under the Apache License,
+ Version 2.0 (the "License"); you may not use this file except
+ in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied. See the License for the
+ specific language governing permissions and limitations
+ under the License.
+"""
+
+import logging
+
+import gspread
+import openai
+import pinecone
+from oauth2client.service_account import ServiceAccountCredentials
+
+from constants import GS_CREDENTIALS_PATH, EMBEDDING_MODEL, SHEET_ID, WORKSHEET_NAME, PINECONE_INDEX_NAME, \
```

Comment:
Import all at once instead of importing one by one

Reply:
fixed in [ba63630](https://github.com/wso2/choreo-examples/pull/74/commits/ba6363086b83e46cd93d7efb70ebaff83e5b9343)